FRCore/FRCore/Authenticator/CryptoKey.swift

```diff
@@ -11,40 +11,40 @@
 
 import Foundation
 import CommonCrypto
-import FRCore
 import CryptoKit
 
+
 ///Helper struct to generate and sign the keys
-struct KeyAware {
-    private var userId: String
+public struct CryptoKey {
+    private var keyId: String
     var timeout = 60
     private var keySize = 256
```

Comment:
Could we make this public in case any customer want to make this key size bigger?

Reply:
We are using Secure Enclave here for storing the keys and Secure Enclave only supports 256-bit elliptic curve keys.